### DISCUSSION TITLE
Prisma Cloud

### DISCUSSION THREAD

**Discussion by emrekrt1655** [prisma/prisma#18270] [Q&A] [answered]
Hi, Is there any way to use Prisma in cloud freely? A few freiend are developing sth together and we want to use a common database. We created node server application with prisma and we want to move the database to the cloud.

**Comment by nurul3101** [comment, marked as answer]
Hello @emrekrt1655 👋 

Prisma Data Platform (http://cloud.prisma.io/) has a free tier that you can use. Details can be found on our [pricing](https://www.prisma.io/pricing) page. On a sidenote, Prisma Data Platform is not a database provider, you can get more information about offerings of Prisma Data Platform [here](https://www.prisma.io/data-platform).
